packages/salesforcedx-utils-vscode/src/helpers/paths.ts

```diff
@@ -162,6 +162,5 @@ export const projectPaths = {
   toolsFolder,
   lwcTestResultsFolder,
   relativeStateFolder,
-  relativeToolsFolder,
-  TOOLS
+  relativeToolsFolder
```

Comment:
wanted projectPaths to be only functions so removed the constant and exported it stand alone above.